src/Transport/RestTransport.php

```diff
@@ -264,4 +288,66 @@ private function getCallOptions(array $options)
 
         return $callOptions;
     }
+
+    /**
+     * @param RequestInterface $request
+     *
+     * @return array
+     */
+    private function logRequest(RequestInterface $request)
```

Comment:
This logic could go in [HttpUnaryTransportTrait](https://github.com/googleapis/gax-php/blob/main/src/Transport/HttpUnaryTransportTrait.php) so that it's also available for [GrpcFallbackTransport](https://github.com/googleapis/gax-php/blob/main/src/Transport/GrpcFallbackTransport.php)